src/repl/scala/tools/nsc/interpreter/IMain.scala

```diff
@@ -710,6 +711,30 @@ class IMain(val settings: Settings, parentClassLoaderOverride: Option[ClassLoade
     }
   }
 
+  @inline private final def tyParens[T](ts: Iterable[T]): String       = ts.mkString("[", ", ", "]")
+  @inline private final def implicitParens[T](ts: Iterable[T]): String = ts.mkString("(implicit ", ", ", ")")
+  @inline private final def parens[T](ts: Iterable[T]): String         = ts.mkString("(", ", ", ")")
+
+  private def methodTypeAsDef(tp: Type): String = {
+
+    def withoutImplicit(sym: Symbol): Symbol = sym.cloneSymbol(sym.owner, sym.flags - Flag.IMPLICIT)
```

Comment:
`sym.flags &~ IMPLICIT` is a bit more idiomatic, I think (this works, but obscures the fact that it's bit-twiddling that we're doing)

Reply:
Yeah this makes sense